R/FineTuningAndPrediction.R

```diff
@@ -197,22 +204,24 @@ trainModel <- function(pretrainedModelFolder,
                        personSequenceFolder,
                        fineTunedModelFolder,
                        trainingSettings,
-                       modelType) {
+                       predictionHead) {
```

Comment:
I thought `predictionHead` was more descriptive for this variable than `modelType`. Also `modelType` is already used in the PLP so it could be confusing.